File: src/components/JSONHandler/src/DeleteCommand.cpp
#include "JSONHandler/DeleteCommand.h"
#include "JSONHandler/RPC2Marshaller.h"

using namespace RPC2Communication;

DeleteCommand::DeleteCommand()
:RPC2Request( RPC2Marshaller::METHOD_DELETECOMMAND_REQUEST )
{}
 
DeleteCommand::~DeleteCommand()
{}

void DeleteCommand::setCmdId(int id)
{
    cmdId = id;
}
        
int DeleteCommand::getCmdId() const
{
    return cmdId;
}

File: mobile/android/SyncProxyAndroid/src/com/ford/syncV4/proxy/rpc/DeleteCommand.java
package com.ford.syncV4.proxy.rpc;

import java.util.Hashtable;

import com.ford.syncV4.proxy.RPCRequest;
import com.ford.syncV4.proxy.constants.Names;

public class DeleteCommand extends RPCRequest {

    public DeleteCommand() {
        super("DeleteCommand");
    }
    public DeleteCommand(Hashtable hash) {
        super(hash);
    }
    public Integer getCmdID() {
        return (Integer) parameters.get( Names.cmdID );
    }
    public void setCmdID( Integer cmdID ) {
        if (cmdID != null) {
            parameters.put(Names.cmdID, cmdID );
        }
    }
}